R/Nancy Clarin/excel_to_csv.R

```diff
@@ -0,0 +1,3 @@
+slx <- read_excel("/home/clarinnancy/tickets/Hamilton2016/Alaska_place_time_16b (1).xls")
```

Comment:
you should always include your library calls (package dependencies) at the top of every R script that you write. This way your code is more reproducible (you can run this script on its own) and other people can run it easily

Reply:
additionally - I would fix the (1) on your filenames. It is not good practice to name files with spaces and parentheses (even if you downloads folder does it for you all the time). I always am in the practice of renaming files once I upload them onto the server.